src/udev/udev-event.c

```diff
@@ -1041,6 +1041,10 @@ int udev_event_execute_rules(UdevEvent *event,
         if (r < 0)
                 return log_device_debug_errno(dev, r, "Failed to update database under /run/udev/data/: %m");
 
+        r = update_devnode(event);
```

Comment:
maybe add a comment here, why this is done again. The uninitiated onlooker might otherwise get the impression this was a copy/paste error somehow